inc/extras.php

```diff
@@ -27,3 +27,44 @@ function zillah_body_classes( $classes ) {
 	return $classes;
 }
 add_filter( 'body_class', 'zillah_body_classes' );
+
+
+/**
+ * Slider Posts to body Posts on mobile.
+ */
+function zillah_slider_to_posts() {
+
+    $zillah_categ = get_theme_mod('zillah_home_slider_category');
+    $args = array(
+        'cat'                   => $zillah_categ,
```

Comment:
@abaicus can you please check here if the category is not empty? I think this is causing a notice sometimes and the query should run only when a category is selected